mirage/views/identifier.ts

```diff
@@ -0,0 +1,8 @@
+import { faker, HandlerContext, Schema } from 'ember-cli-mirage';
+
+export function identifierCreate(this: HandlerContext, schema: Schema) {
+    return schema.identifiers.create({
+        ...this.normalizedRequestAttrs('identifier'),
+        value: faker.fake('10.5555/{{company.bsNoun}}'),
```

Comment:
Could this be handled by the mirage factory instead, or is there something I'm missing?